src/libraries/System.Runtime/tests/System.Runtime.Tests/System/ArrayTests.cs

```diff
@@ -1950,10 +1946,7 @@ public void CreateInstance_NotSupportedType_ThrowsNotSupportedException(Type ele
         }
 
         [Theory]
-        [InlineData(typeof(void))]
         [InlineData(typeof(GenericClass<>))]
-        // not using any by-ref type here as MakeArrayType throws for them, same goes for Type.GetType("SomeByRef[]")	
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/94086", typeof(PlatformDetection), nameof(PlatformDetection.IsMonoRuntime))]
```

Comment:
if we remove the link to the issue it will be harder to reproduce for the Mono Team

How about leaving it like this?

```cs
// [InlineData(typeof(void))] fails on Mono: https://github.com/dotnet/runtime/issues/94086
```

Reply:
My plan is to disallow arrays of System.Void in a follow up PR that will fix the Mono issue as a side-effect: https://github.com/dotnet/runtime/pull/94835

Reply:
@jkotas sounds like a much better plan!